scripts/generate-sdf.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs')
```

Comment:
nit, but add `#!/usr/bin/env node` as first line?

Reply:
Happy for you to add that. Would also need to make it executable. I tend towards keeping scripts without a hashbang when they are not being published as scripts to run outside the module - just to always run them with `node ./script.js`

Reply:
good point! not necessary in this case